frontend/components/Domain/ShoppingList/ShoppingListItem.vue

```diff
@@ -59,7 +59,7 @@
                 open-delay="200"
                 transition="slide-x-reverse-transition"
                 density="compact"
-                right
+                location="end"
```

Comment:
Switched this from `right` to `end` because the only difference is for RTL languages ("right" is _always_ right, whereas "end" is right for LTR languages, and left for RTL languages).